src/algorithms/calorimetry/CalorimeterClusterRecoCoG.cc

```diff
@@ -175,6 +175,7 @@ std::optional<edm4eic::Cluster> CalorimeterClusterRecoCoG::reconstruct(const edm
     m_log->debug("hit energy = {} hit weight: {}", hit.getEnergy(), weight);
     auto energy = hit.getEnergy() * weight;
     totalE += energy;
+    cl.addToHits(hit);
     if (energy > maxE) {
     }
```

Comment:
OT but what's this doing here???

Reply:
Dropped by @sly2j in https://eicweb.phy.anl.gov/EIC/juggler/-/commit/fabb1ece72b00fbabd8062bf7b775344670abb49#8c0365ec766abf3cb50e95d734f31066c8853df3_173_171 (https://eicweb.phy.anl.gov/EIC/juggler/-/merge_requests/392). This might actually have been a mistake that snuck in via an unrelated MR.

Reply:
Ok, so this originates in using the block with maximum energy deposition in the cluster to define the `time` of the cluster. At some point (in this MR above), the maximum energy selection was removed, and we ended up writing the `time` based on the first hit in the protocluster. So, right now (and since then) the cluster time is not time of the maximum energy hit anymore.

Reply:
Deferred to https://github.com/eic/EICrecon/issues/1225.